app/js/actions/AuthActions.js

```diff
@@ -40,9 +40,13 @@ export function logoutUser() {
 export function loginUser() {
   return dispatch => {
     dispatch(beginAjaxCall());
-    return webAPI(`/login`, 'POST')
+    return webAPI(`/users`, 'GET')
```

Comment:
I am not sure i understand what i would need to refactor here.I need further clarifications here.